_includes/manuals/1.9/7.3_getting_help.md

```diff
@@ -8,3 +8,24 @@ Mailing lists are available via Google Groups. Much like IRC, we have a general
 
 * [foreman-users](https://groups.google.com/forum/?fromgroups#!forum/foreman-users)
 * [foreman-dev](https://groups.google.com/forum/?fromgroups#!forum/foreman-dev)
+
+### Gathering information
+In order to troubleshoot and get relevant data use foreman-debug which collects information about your OS, Foreman and related components.
+If you installed from packages, the command is available to root:
+
+    # foreman-debug
+
+If you installed from git, you can find it in the Foreman directory:
+
+    # script/foreman-debug
+
+If you run it without any options, it will collect data, filter out possible
+passwords or tokens and create a tarball which can be safely handed over to
+us.
+
+To upload the tarball to our public server via rsync use:
+
+        foreman-debug -u
```

Comment:
NItpick, please match indentation of the commands above (four spaces) and add the `#` to indicate a root prompt.